images/php-fpm/8.1.Dockerfile

```diff
@@ -38,8 +38,8 @@ ENV TMPDIR=/tmp \
 COPY check_fcgi /usr/sbin/
 COPY entrypoints /lagoon/entrypoints/
 
-COPY php.ini /usr/local/etc/php/
-COPY 00-lagoon-php.ini.tpl /usr/local/etc/php/conf.d/
+RUN cp "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/conf.d/php.ini"
+COPY 00-lagoon-php.ini.tpl "$PHP_INI_DIR/conf.d/"
```

Comment:
8.1 is copying these assets into the `$PHP_INI_DIR/conf.d` folder - the others into `$PHP_INI_DIR/`

Reply:
Nice catch!